### DISCUSSION TITLE
4.6.15

### DISCUSSION THREAD

**Discussion by ywmoyue** [ywmoyue/biliuwp-lite#316] [Announcements]
#### FIXED
- 修复4.6.14引入Brotil算法库后直播弹幕不可用问题

<hr /><em>This discussion was created from the release <a href='https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.6.15'>4.6.15</a>.</em>

**Comment by jnbadnsodkns** [comment]
更新到4.6.15还是存在闪退的情况，这是日志
[20230917.log](https://github.com/ywmoyue/biliuwp-lite/files/12642129/20230917.log)

**Comment by GD-Slime** [comment]
> 更新到4.6.15还是存在闪退的情况，这是日志 [20230917.log](https://github.com/ywmoyue/biliuwp-lite/files/12642129/20230917.log)

如果从日志来看的话是因为kgithub请求失败导致的。 kgithub最近换了域名，我等会改一下。 #318

**Reply by GD-Slime** [reply]
当然，你可以试试先换到ghproxy 后会不会还出现闪退。 @FENGRANMOXIA

**Reply by ywmoyue** [reply]
@GD-Slime 我感觉这里请求失败似乎不应该会导致闪退，因为有catch捕获异常，我在pr上给加个空引用处理吧
@FENGRANMOXIA 在设置其他页中更新请求地址换到 ghproxy 后还会出现闪退吗

**Comment by ywmoyue** [comment]
> 对了，直到现在的版本还存在一个问题，收藏夹不显示16位uid新用户发布的视频，不知这个有没有办法解决

@FENGRANMOXIA 这个问题我新开了一个帖子在那讨论，在这里不方便管理记录和后续查询追踪

https://github.com/ywmoyue/biliuwp-lite/discussions/322

**Comment by ywmoyue** [comment]
@FENGRANMOXIA 试试这个issue下提到的操作步骤，看他的描述和你的情况很像
